client/my-sites/checkout/src/components/akismet-pro-quantity-dropdown/index.tsx

```diff
@@ -271,6 +288,33 @@ export const AkismetProQuantityDropDown: FunctionComponent< AkismetProQuantityDr
 		]
 	);
 
+	const { quantity, currency, item_subtotal_integer } = responseCart.products[ 0 ];
+	const itemSubtotalQuantityOneInteger = item_subtotal_integer / ( quantity ?? 1 );
+
+	const actualAmountQuantityOneDisplay = formatCurrency( itemSubtotalQuantityOneInteger, currency, {
+		isSmallestUnit: true,
+		stripZeros: true,
+	} );
+
+	const actualAmountDisplay = formatCurrency( item_subtotal_integer, currency, {
+		isSmallestUnit: true,
+		stripZeros: true,
+	} );
+
+	const currentOptionPriceDisplay = translate(
+		'%(quantity)d licenses X %(actualAmountQuantityOneDisplay)s per license = %(actualAmountDisplay)s',
```

Comment:
Do we need a translators comment for this string?

Reply:
Yes, good idea, wouldn't hurt! 👍